package.json

```diff
@@ -51,7 +54,9 @@
   },
   "scripts": {
     "test": "mocha",
-    "compile": "babel src --out-dir dist",
+    "compile:cjs": "babel src --out-dir dist",
+    "compile:es": "cross-env BABEL_ENV=es babel src --out-dir es",
+    "compile": "npm-run-all --parallel compile:cjs compile:es",
```

Comment:
It'll run `yarn compile:cjs` and `yarn compile:es`, instead of `npm run compile:cjs` and `npm run compile:es`.
If you do `"compile": "npm run compile:cjs && npm run compile:es"`, and then run `yarn compile`, the `npm run compile:cjs` line will be run using npm. `npm-run-all` detects that it's being run with Yarn and uses it.